packages/vite/src/node/server/index.ts

```diff
@@ -902,19 +916,24 @@ export async function _createServer(
     }
   })
```

Comment:
We should skip the `cachedTransformMiddleware` above in full bundle mode.

Reply:
Done